apps/src/templates/teacherHomepage/Announcement.jsx

```diff
@@ -61,12 +60,17 @@ const Announcement = React.createClass({
   },
 
   render() {
-    const { heading, buttonText, description, link } = this.props;
+    const { heading, buttonText, description, link, image } = this.props;
+    const filenameToImgUrl = {
+      "navcard-placeholder.png": require('../../../static/navcard-placeholder.png'),
```

Comment:
I think maybe I asked before if it's easier to use an absolute path at this point...